invisible-wallet/typescript/package.json

```diff
@@ -1,9 +1,12 @@
 {
   "devDependencies": {
-    "typescript": "^5.0.4"
+    "typescript": "^5.2.2"
   },
   "dependencies": {
-    "@mysten/sui.js": "^0.41.0",
+    "@mysten/sui.js": "^0.45.1",
+    "@shinami/clients": "^0.6.0",
```

Comment:
There's a newer version.

Reply:
Updated both `@mysten/sui.js" and "@shinami/clients" for both the invisible wallet and gas station. Will be reflected in my next push.